verifier.go

```diff
@@ -465,9 +466,34 @@ func validateRPCDataType(
 	return nil
 }
 
+func getPackageName(datatypeName string) string {
```

Comment:
A comment explaining the assumptions and the result if the assumption is not true would be useful.